### ISSUE TITLE
Atomist :: Rug -> Elm

### ISSUE THREAD

**Issue by newswim** [dwyl/learn-elm#48]
Jessica Kerr, aka @jessitron [[blog](http://blog.jessitron.com)], is working on a really neat usage of _Rug_ called **Atomist** which is sort of like Rails or Yeoman, for an Elm project. 🤷‍♂️  

Rug files are usually called "editors", you can see some [here](https://github.com/jessitron/elm-rugs/tree/master/.atomist/editors).

However, Atomist will act like a pair-programming bot, you can tell it to add new features, make code amendments and automate a lot of tasks, all from *_Slack_*.

**It's still in the `early stages`**, but she's just started working on it [full-time](http://blog.jessitron.com/2017/01/its-atomist-time.html). Atomist/Rug could give us a way to build out Elm projects much faster. It aims to alleviate some of the repetitive tasks like having to update multiple files when adding a new feature. From the looks of it, I would postpone adding much of anything about it to the guide, at least until they have a stable runway, but it looks very promising! 👀

### [Why and What is Rug?](http://docs.atomist.com/rug/why-and-what-is-rug)

TL;DR :: Rails for Elm, yo!
